compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -77,18 +78,22 @@ class ComposeScene internal constructor(
      * @param coroutineContext Context which will be used to launch effects ([LaunchedEffect],
      * [rememberCoroutineScope]) and run recompositions.
      * @param density Initial density of the content which will be used to convert [dp] units.
+     * @param layoutDirection Initial layout direction of the content.
      * @param invalidate Callback which will be called when the content need to be recomposed or
      * rerendered. If you draw your content using [render] method, in this callback you should
      * schedule the next [render] in your rendering loop.
      */
+    @ExperimentalComposeUiApi
     constructor(
         coroutineContext: CoroutineContext = Dispatchers.Unconfined,
         density: Density = Density(1f),
+        layoutDirection: LayoutDirection = LayoutDirection.Ltr,
```

Comment:
Why do we need 2 more public constructors? I mean, I know, keeping old ones required for compatibility, but what the diference in behavior if I'd use old constructor and set `layoutDirection` via setter on next line?

Reply:
Igor asked me to do it this way.

Reply:
If we could remove density from the constructor, I would agree. But we can't, and having density there but not layoutDirection is a bit inconsistent.

Reply:
I don't have big preference here, let's @m-sasha decide then.

P.S. We also have ImageComposeScene, and we add properties to constructor there by design (to simplify construction code)